go.mod

```
@@ -220,6 +220,7 @@ replace (
 	github.com/cosmos/cosmos-sdk => github.com/evmos/cosmos-sdk v0.46.7-ledger
 	// use Evmos geth fork
 	github.com/ethereum/go-ethereum => github.com/evmos/go-ethereum v1.10.26
+	github.com/evmos/ethermint v0.20.0 => github.com/notional-labs/ethermint v0.6.1-0.20230107054512-4e67a8e3ddff
```

This is currently breaking our upgrade tests (`make test-upgrade`)
```suggestion
	github.com/evmos/ethermint v0.20.0 => github.com/notional-labs/ethermint v0.6.1-0.20230107054512-4e67a8e3ddff
```

---

hmmm yeah the forked version are from the main brach of ethermint so totally aggreed with you if it break the test. either i'm gonna bump the ibc-go on the release ethermint v0.20 or wait for newer release that use ibc v6 since that PR was merged on ethermint

---

@MalteHerrmann hey i just switch to forked version of ethermint v0.20.0 can you run the test again and let me know if it working or not

---

@sontrinh16 I have addressed the issues on Ethermint and will include a commit to your PR with the relevant changes to make upgrades work as well as replace Ethermint with our current `main` branch.